scripts/ci/package.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Requires packages:
+# devscripts cmake ninja git
+
+TOP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../../" && pwd )"
+
+if [ ! -d "tmp-cmake" ]; then
+mkdir tmp-cmake
+cd tmp-cmake
+wget https://cmake.org/files/v3.5/cmake-3.5.2-Linux-x86_64.tar.gz
+tar xf cmake-3.5.2-Linux-x86_64.tar.gz
+cd cmake-3.5.2-Linux-x86_64/bin/
+export PATH=`pwd`:$PATH
+else
+cd tmp-cmake/cmake-3.5.2-Linux-x86_64/bin
+export PATH=`pwd`:$PATH
+fi
```

Comment:
You won't need this whole bit if you just add cmake to your Build-Depends: in debian/control (see corresponding line note)